_faqs/01-08-is-opensearch-wire-compatible-with-elasticsearch.markdown

```diff
@@ -1,5 +0,0 @@
----
-category: General
-question: Is OpenSearch wire-compatible with Elasticsearch?
----
-Yes. OpenSearch is a fork of open source Elasticsearch 7.10. As such, it provides the same REST APIs for ingest, search, and management. The query syntax and responses are also the same. In addition, OpenSearch can use indices from Elasticsearch versions 6.0 up to 7.10. We also aim to support the existing Elasticsearch clients that work with Elasticsearch 7.10.
```

Comment:
Which FAQ is this duplicative of in the ones added? I think we should still highlight that REST APIs remain the same, search APIs remain the same, query responses etc since wire compatibility also has to do with tools being used with OpenSearch.

Reply:
I've restored this FAQ and provided a more details link.